src/Form/Field/ObjectChoiceList.php

```diff
@@ -34,33 +31,37 @@ public function __construct(
         $this->withWarning = $withWarning;
     }
 
-    public function getTypes()
+    public function getTypes(): string
     {
         return $this->types;
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```

Comment:
remove {@inheritDoc} ?

Reply:
Referering to: Symfony\Component\Form\ChoiceList\ChoiceListInterface methods